### ISSUE TITLE
Show an onboarding or help screen on first startup

### ISSUE THREAD

**Issue by phkaeser** [phkaeser/wlmaker#131]
To explain...
* how to exit
* how to run other applications
* how to configure it.

Since the configured keyboard shortcut is too obscure for starters.

**Comment by alexmyczko** [comment]
I dont think it should have a help screen showing up without being asked for. But I would like a contect menu like the original (nextstep) or wmaker. 

right mouse button click on desktop

```
Title
Quit            ^Q
Run…     alt-space
Settings
```
